modules/storages/spec/features/admin_storages_spec.rb

```diff
@@ -31,12 +31,65 @@
 require 'spec_helper'
 require_module_spec_helper
 
-RSpec.describe 'Admin storages', :storage_server_helpers, js: true do
+RSpec.describe 'Admin storages',
+               :js,
+               :storage_server_helpers do
   let(:admin) { create(:admin) }
 
   before { login_as admin }
 
-  it 'creates, edits and deletes storages', webmock: true do
+  describe 'File storages list', with_flag: { storage_primer_design: true } do
+    context 'with storages' do
+      let(:complete_storage) { create(:nextcloud_storage_with_local_connection) }
+      let(:incomplete_storage) { create(:nextcloud_storage) }
+
+      before do
+        complete_storage
+        incomplete_storage
+      end
+
+      it 'renders a list of storages' do
+        visit admin_settings_storages_path
+
+        expect(page).to have_css('.op-storage-list--name', text: complete_storage.name)
```

Comment:
We have the patter to not test on classes any more as you'd have to adapt all tests if you change a class name. Instead we have a data attribute called `data-test-selector` which you set on those elements that you need, and then test for it. This selector is independent of the class and idealy not touched any more in the future.